setup.py

```diff
@@ -127,7 +127,8 @@ def run(self):
         cmake_args += ['-DCMAKE_BUILD_TYPE=' + cfg]
 
         # Assuming Makefiles
-        build_args += ['--', f'-j{multiprocessing.cpu_count()}']
+        if get_os_name() != 'win':
```

Comment:
sorry I'm not familiar with windows, why do we want to specialize this? ;)

Reply:
This is simply because MSBuild does not support "-j{number}" switch for multiprocessor build.